Adjust short answer question No.

CiSong10 opened the pull request.

CiSong10 committed:
Adjust short answer question No.

CiSong10 committed:
Clarify the short answer 3

Students tend to treat "this network" in Short answer 3 as the very
simple example network above -- which do not need to "guess" the
distances.

Changed "this network" to specify the network being the _A Storm of Swords_ network `G_asoiaf`

packerliu commented on 1_Network_Structure.ipynb:
```diff
@@ -659,10 +659,10 @@
     "\n",
     "#### Short Answer 3\n",
-    "Write a few sentences about the distance (number of edges) between nodes in this network, considering the following. \n",
+    "Write a few sentences about the distance (number of edges of the shortest path) between nodes in the _A Storm of Swords_ network `G_asoiaf`, considering the following. \n",
```
Yes. It is good to indicate the network is for the "_A Storm of Swords_". 
I would not add "of the shortest path" since it duplicates the definition of the distance. 
The "number of edge" was added there to distinguish it from the other option, the "number of intermediate node".

packerliu commented on 1_Network_Structure.ipynb:
```diff
@@ -659,10 +659,10 @@
-    "2. How long would you guess the shortest path is between the two most distance nodes? \n",
+    "1. How long would you guess the typical distance between two nodes is? \n",
+    "2. How long would you guess the distance is between the two most distance nodes? \n",
     "    \n",
```
I suggest not to change these two places from "shortest path" to "distance".  
They intent to highlight the definition of the "distance" by the length of the "shortest path" not any other paths.

CiSong10 committed:
Revert "Clarify the short answer 3"

This reverts commit f17116537c854c2429452499b6b082fcb7ccdafc.

CiSong10 committed:
Clarify short answer 3

Students tend to treat "this network" in Short answer 3 as the very simple example network above
-- which do not need to "guess" the distances.

Changed "this network" to specify the network being the _A Storm of Swords_ network `G_asoiaf`

CiSong10 commented:
Changed according to @packerliu's request
